lib/src/ledger/ledger_gatt_gateway.dart

```diff
@@ -251,12 +247,11 @@ class LedgerGattGateway extends GattGateway {
       final services = await UniversalBle.discoverServices(ledger.device.id);
 
       final targetUuid = serviceId.toLowerCase();
-      final targetUuidWithSuffix = serviceIdWithSuffix.toLowerCase();
 
       final foundService = services.firstWhere(
         (s) =>
             s.uuid.toLowerCase() == targetUuid ||
```

Comment:
I believe this line is not needed anymore. The next one should already cover this too.